charts/manifest-deployer/templates/clusterrole.yaml

```diff
@@ -64,4 +64,15 @@ rules:
     - get
     - watch
     - list
+
+- apiGroups:
```

Comment:
Unfortunately because this clusterrole was never really being used. When using the agent and the automatic deployer management, these clusterroles have been created by the go controller. Now that we are installing the deployer via helm chart, this particular permission was missing.